x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authz/store/NativeRolesStoreTests.java

```diff
@@ -300,10 +300,17 @@ private ClusterState getClusterStateWithSecurityIndex() {
             RecoverySource.ExistingStoreRecoverySource.INSTANCE,
             new UnassignedInfo(Reason.INDEX_CREATED, "")
         );
-        IndexShardRoutingTable table = new IndexShardRoutingTable.Builder(new ShardId(index, 0)).addShard(
-            shardRouting.initialize(randomAlphaOfLength(8), null, shardRouting.getExpectedShardSize()).moveToStarted()
-        ).build();
-        RoutingTable routingTable = RoutingTable.builder().add(IndexRoutingTable.builder(index).addIndexShard(table).build()).build();
+        RoutingTable routingTable = RoutingTable.builder()
+            .add(
+                IndexRoutingTable.builder(index)
+                    .addIndexShard(
+                        new IndexShardRoutingTable.Builder(new ShardId(index, 0)).addShard(
```

Comment:
Can we add `IndexShardRoutingTable.builder(ShardId)` and use instead? This looks a bit odd here given the two different styles.

Reply:
++ done